src/drivers/MySQLDriver.ts

```diff
@@ -19,27 +18,30 @@ export class MySQLDriver implements IDriver {
     }
 
     private checkConnection(): void {
-        if (this.conn == null)
+        if (!this.conn) {
             throw new Error("MySQL not connected to the database");
+        }
     }
 
     async connect(): Promise<void> {
-        this.conn = (await this.mysql.createPool(this.config)) as any;
+        this.conn = await this.mysql.createPool(this.config);
     }
 
     async prepare(table: string): Promise<void> {
         this.checkConnection();
 
-        await this.conn?.query(
+        await this.conn.query(
             `CREATE TABLE IF NOT EXISTS ${table} (ID VARCHAR(255) PRIMARY KEY, json TEXT)`
         );
     }
 
     async getAllRows(table: string): Promise<{ id: string; value: any }[]> {
         this.checkConnection();
 
-        const results = await this.conn?.query(`SELECT * FROM ${table}`);
-        return results.map((row: any) => ({
+        const results = await this.conn.query<RowDataPacket[]>(
+            `SELECT * FROM ${table}`
+        );
+        return results.map((row) => ({
```

Comment:
Here eslint errors out because row has implicit any type. The any was there to prevent this error (I want to keep this eslint config so it's not forgotten to add types)
```suggestion
        return results.map((row: any) => ({
```